ci/format_pre.sh

```diff
@@ -50,9 +57,9 @@ CURRENT=extensions
 bazel run "${BAZEL_BUILD_OPTIONS[@]}" //tools/extensions:extensions_check
 
 if [[ "${#FAILED[@]}" -ne "0" ]]; then
-    echo "TESTS FAILED:" >&2
+    echo "##[error]: TESTS FAILED:" >&2
```

Comment:
looks good. Suggest making a bash variable ERROR_PREFIX and using it here and below. When you set BASH_PREFIX you can document why the syntax is desired.

Reply:
sure - ill update...